### ISSUE TITLE
Revisions Compare Form Accessibility

### ISSUE THREAD

**Issue by seshelby** [GetDKAN/dkan#3043]
The compare form includes radio buttons with empty labels and are not accessible to screen reader users. The radio buttons should include aria-label attributes.

While viewing a dataset, click on the revisions button and review he html code for the radio buttons

**Comment by janette** [comment]
This would require a patch to [diff](https://www.drupal.org/project/diff), if you get something working let us know, otherwise we will add it to our backlog, however we are focused on the Drupal 8 version of DKAN at the moment and may not get to this.
